### DISCUSSION TITLE
pdf的关键信息抽取报错

### DISCUSSION THREAD

**Discussion by dogrepairditch** [PaddlePaddle/PaddleOCR#12381] [Q&A] [answered]
Traceback (most recent call last):
  File "kie/predict_kie_token_ser.py", line 181, in <module>
    main(parse_args())
  File "kie/predict_kie_token_ser.py", line 149, in main
    img = img[:, :, ::-1]
TypeError: 'NoneType' object is not subscriptable

当抽取pdf关键信息时，报错如上
启动命令如下：

python3 kie/predict_kie_token_ser.py \
  --kie_algorithm=LayoutXLM \
  --ser_model_dir=../inference/ser_vi_layoutxlm_xfund_infer \
  --image_dir=/root/paddleocr/ewerv .pdf \
  --ser_dict_path=../train_data/XFUND/class_list_xfun.txt \
  --vis_font_path=/root/paddleocr/doc/fonts/simfang.ttf  \
  --ocr_order_method="tb-yx"

请问这种应该怎么办呢？

**Comment by tink2123** [comment, marked as answer]
predict_kie_token_ser.py 脚本不支持上传pdf，只能上传图片或者文件夹
